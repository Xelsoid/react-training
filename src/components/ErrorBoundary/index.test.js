import React from 'react';
import ComponentExample from './index';

describe('ErrorBoundary test', () => {
  it('renders correctly', () => {
    const component = shallow(<ComponentExample />);
    expect(component).toMatchSnapshot();
  });
});
