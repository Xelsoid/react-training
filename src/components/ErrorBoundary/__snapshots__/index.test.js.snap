// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ErrorBoundary test renders correctly error happened  1`] = `
<div
  className="error-boundary-wrapper"
>
  <div
    className="error-boundary"
  >
    Something went wrong. layout is broken.
  </div>
</div>
`;

exports[`ErrorBoundary test renders correctly no error happened 1`] = `<NotEmptyComponent />`;
